CHANGELOG.md

```diff
@@ -77,6 +77,7 @@
 * [BUGFIX] Querier: fix issue where queries can return incorrect results if a single store-gateway returns overlapping chunks for a series. #8827
 * [BUGFIX] Querier: do not return `grpc: the client connection is closing` errors as HTTP `499`. #8865 #8888
 * [BUGFIX] Compactor: fix a race condition between different compactor replicas that may cause a deleted block to be still referenced as non-deleted in the bucket index. #8905
+* [BUGFIX] Querier: fix issue where some native histogram-related warnings were not emitted when `rate()` was used over native histograms. #8918
```

Comment:
@bboreham I haven't added a changelog entry for https://github.com/prometheus/prometheus/pull/14584 given the bug was introduced in https://github.com/prometheus/prometheus/pull/14396 and that is also part of the next release. If you disagree, let me know and I'll add something.